### DISCUSSION TITLE
error maj v.0.5.3 help me please

### DISCUSSION THREAD

**Discussion by 0xEmpereur** [eqlabs/pathfinder#1028] [General]
Hi after doing starknet shift 5.0.3 as soon as I try to relaunch the node its puts me its error message how to set it please?
![image](https://user-images.githubusercontent.com/130609589/231657226-d4e29fb1-7956-47b6-a917-b7955ac76818.png)

**Comment by Mirko-von-Leipzig** [comment]
I think you're missing the [step to update the python](https://github.com/eqlabs/pathfinder/blob/main/doc/install-from-source.md#python-dependencies) dependencies:

> Next, update the python dependencies. First enable your python virtual environment (if you are using one). For our example installation this would be:
> ```
> source ./py/.venv/bin/activate
> ```
> and then update:
> ```
> PIP_REQUIRE_VIRTUALENV=true pip install -e py/.[dev]
> ```

**Reply by 0xEmpereur** [reply]
Hi after trying your solu
![image](https://user-images.githubusercontent.com/130609589/231738404-ae10a4be-5274-478b-b5c3-388fdce666d2.png)
tion your error message appears and as soon as I try to restart the node the first error message did not move

**Reply by Mirko-von-Leipzig** [reply]
Ah okay its because you are already in the `py` directory. Try this slight modification instead (from inside `py` folder).
```
PIP_REQUIRE_VIRTUALENV=true pip install -e .[dev]
```

**Reply by 0xEmpereur** [reply]
thank you very much for her rematch🤩
